lib/options-from-context.js

```diff
@@ -0,0 +1,17 @@
+// Copyright IBM Corp. 2013,2016. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+module.exports = buildOptionsFromRemotingContext;
+
+function buildOptionsFromRemotingContext(ctx) {
+  var accessToken = ctx.req.accessToken;
+  var options = {
```

Comment:
+1, minxin is great. Actually, most of my scenarios need currentUser with full property, not only currentUserId. someone maybe another.